watchmal/dataset/pointnet/pointnet_mpmt_dataset.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
+import torch
+
+from watchmal.dataset.h5_dataset import H5Dataset
+from watchmal.dataset.pointnet import transformations
+import watchmal.dataset.data_utils as du
+
+barrel_map_array_idxs = np.array([ 6,  7,  8,  9, 10, 11,  0,  1,  2,  3,  4,  5, 15, 16, 17, 12, 13, 14, 18], dtype=np.int16)
```

Comment:
Not essential, but do we want to move these to a shared location given that they're used by both mpmt dataset classes?

Reply:
Yeah, I was thinking this. I'm not 100% sure that the pointnet mPMT needs to keep it at all... I'm having a hard time visualising whether the flipping the barrel PMTs within mPMTs is necessary when you have it in 3D rather than the 2D image. It might be that it's not actually needed for the 3D because the inconsistency in the 2D image is just due to the choice of angle around the tank where you unwrap it. I'll do some validation on the mPMT version of pointnet and if needs to stay then we can move it somewhere shared I guess.